### ISSUE TITLE
Do not require "to" address

### ISSUE THREAD

**Issue by moonstreamis** [muquit/mailsend-go#20]
Great product simple and straight forward,  One quirk so far.  It requires a to address.  If you have a bcc or a cc, this should not be required.  I cannot see anyway of getting round this.  Thanks.

**Comment by muquit** [comment]
Yes technically there is no requirement for **To** field if at least a Cc or a Bcc address is specified. However, the way it is designed, **To** is required and will require some work to remove this restriction. Unfortunately I do not have free time to do that at this time. If anyone sends a pull request, I will consider accepting it.

**Comment by Bilbo47** [comment]
-t parameter should also be ignored and not-required when -list parameter is specified. In version 1.0.10, specifying a to-address in both -t and -list file will send two separate messages to the one same address; one message of the two will have the to-name portion from the -list file.
